src/app/modules/consultations/consultation-profile/consultation-questionnaire/consultation-questionnaire.component.ts

```diff
@@ -422,7 +422,7 @@ export class ConsultationQuestionnaireComponent implements OnInit, AfterViewInit
     const consultationResponse =  {
       consultationId: this.profileData.id,
       satisfactionRating : this.responseFeedback,
-      visibility: this.responseVisibility && this.currentUser?.isVerified ? "shared" : "anonymous",
+      visibility: this.responseVisibility,
```

Comment:
This will allow unverified users to submit public responses, I don't think this is what the requested change was